sources/packages/api/src/route-controllers/esdc-integration/e-cert-integration.system.controller.ts

```diff
@@ -100,6 +106,20 @@ export class ECertIntegrationController extends BaseController {
     }));
   }
 
+  /**
+   * Send provincial daily disbursement information to FIN.
+   * @param processDate Batch run date.
+   * @returns Summary details of processing.
+   */
+  @Post("process-provincial-daily-disbursements")
+  async processProvincialDailyDisbursements(
+    @Body() payload: DailyDisbursementReportAPIInDTO,
+  ): Promise<string> {
+    return this.disbursementReceiptRequestService.processProvincialDailyDisbursements(
+      new Date(payload.batchRunDate),
```

Comment:
What will happen in this scenario when the batchRunDate is undefined?
I believe that it will lead to wrong processing, can you please check?

Reply:
Thanks for doing the change 👍